packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```diff
@@ -103,7 +103,16 @@ cp -pa .%{gem_dir}/* \
 %{foreman_restart}
 exit 0
 
+%post
+semanage port -a -t http_port_t -p tcp 8006
```

Comment:
Should be `semanage port -a -t http_port_t -p tcp 8006 &> /dev/null || :` to suppress output and always run without error even if SELinux is disabled.